mapit_mysociety_org/templates/account/delete.html

```diff
@@ -14,7 +14,7 @@
         <form method="post" action="{% url "account_delete" %}" autocapitalize="off">
             {% csrf_token %}
             <div class="account-form__field account-form__field--submit">
-                <button type="submit" class="btn">{% trans "Delete My Account" %}</button>
+                <button type="submit" class="btn btn--danger">{% trans "Delete My Account" %}</button>
```

Comment:
Might be unrelated to this PR, but I couldn't find any information about things like the Stripe subscription being cancelled, a pro-rata refund of the current month, etc.